installbuilder/datafiles/Base_DSC.data

```diff
@@ -169,14 +169,16 @@ EOF
 
 chmod u+x $tmpfile
 $tmpfile 1> /dev/null 2> /dev/null
-if [ $? -ne 0 ]; then
+tmpfile_ctypes=$?
+rm $tmpfile
+
+python -c "import ctypes" 1> /dev/null 2> /dev/null
```

Comment:
Whoops, I made a mistake here. The install script **is** creating a file to execute on /tmp [here](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/268/files#diff-2f5547001132d7fa248e64163fcc9e3fR163).

Given this, it's not Python itself, but the way that the script is doing it's thing. Furthermore, while a shell bundle is running, the location of that shell bundle wouldn't be known by the installation script (unless a marker file was created or something with that location).

With all this in mind, I think it's better to just check for `ctypes` inline, and to totally eliminate the creation of the script on `/tmp`, as that's a total artifact of how the postinstall script worked. I see no reason why `NOEXEC` on `/tmp` would be problematic, so if you can safely make the check without the script, that's a good thing.

Can you take a system without `ctypes` installed and be certain the inline method actually works. I believe it will, but best to verify for certain!

When verified, post another review with your changes and I'll sign off.